src/test/kotlin/uk/gov/justice/hmpps/offenderevents/service/UiIntegrationTest.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.hmpps.offenderevents.service
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.boot.web.server.LocalServerPort
+import java.net.URL
+
+class UiIntegrationTest : IntegrationTest() {
+
+  @LocalServerPort
+  private var port: Int = 0
+
+  internal lateinit var baseUrl: String
+
+  @BeforeEach
+  internal fun setupPort() {
+    baseUrl = "http://localhost:$port/"
+  }
+
+  @Test
+  fun `Should Display captured event`() {
+    val message = "/messages/externalMovement.json".readResourceAsText()
+    awsSqsClient.sendMessage(queueUrl, message)
+    `Wait for empty queue`()
+
+    val response = URL("$baseUrl/messages").readText()
+
+    assertThat(response).contains("EXTERNAL_MOVEMENT_RECORD-INSERTED")
```

Comment:
I considered diving into Fluentlenium for these tests, but as this is a dev tool I'm not sure there is any value as I won't be producing a huge test suite.  Therefore the plan is to keep the testing simple and light-touch - just want to prove that the app stands up.

If anyone has better ideas for super simple UI testing then I'm all ears.

Reply:
This looks good for this DEV tool